neo.UnitTests/UT_MemoryPool.cs

```diff
@@ -89,24 +89,28 @@ public void TestSetup()
             _unit.Count.ShouldBeEquivalentTo(0);
         }
 
-        private Transaction CreateRandomHashInvocationTransaction()
+        private Mock<InvocationTransaction> CreateRandomHashInvocationMockTransaction()
         {
-            var tx = new InvocationTransaction();
+            var mockTx = new Mock<InvocationTransaction>();
+            mockTx.CallBase = true;
+            mockTx.Setup(p => p.Verify(It.IsAny<Snapshot>(), It.IsAny<IEnumerable<Transaction>>())).Returns(true);
+            var tx = mockTx.Object;
             var randomBytes = new byte[16];
             _random.NextBytes(randomBytes);
             tx.Script = randomBytes;
             tx.Attributes = new TransactionAttribute[0];
             tx.Inputs = new CoinReference[0];
             tx.Outputs = new TransactionOutput[0];
             tx.Witnesses = new Witness[0];
-            // Force getting the references
-            // Console.WriteLine($"Reference Count: {tx.References.Count}");
-            return tx;
+
+            return mockTx;
         }
 
         private Transaction CreateMockHighPriorityTransaction()
         {
-            var tx = CreateRandomHashInvocationTransaction();
+            var mockTx = CreateRandomHashInvocationMockTransaction();
+            mockTx.SetupGet(p => p.NetworkFee).Returns(Fixed8.FromDecimal(0.001m));
```

Comment:
can fees be random with this limit here?
Maybe it will catch other cases.

Reply:
I mean, random from, for example `[Fixed8.FromDecimal(0.001m), Fixed8.FromDecimal(0.1m)]`

Reply:
They will still be high priority, but with different attached fees